odd-collector/odd_collector/adapters/postgresql/mappers/tables.py

```diff
@@ -1,24 +1,32 @@
 from funcy import lmap, partial, silent
 from odd_collector.logger import logger
-from odd_models.models import DataEntity, DataEntityType, DataSet
+from odd_models.models import Type, DataEntity, DataEntityType, DataSet
 from oddrn_generator import PostgresqlGenerator
 
 from ..models import Table
 from .columns import map_column
 from .metadata import get_table_metadata
 from .views import map_view
+from .types import TYPES_SQL_TO_ODD
 
 
 def map_table(generator: PostgresqlGenerator, table: Table):
     generator.set_oddrn_paths(
         **{"schemas": table.table_schema, "tables": table.table_name}
     )
-
     map_table_column = partial(map_column, generator=generator, path="tables")
+
+    # If table contains vector column we consider it as a vector store, otherwise - ordinary table
+    data_entity_type = DataEntityType.TABLE
+    for c in table.columns:
+        if TYPES_SQL_TO_ODD.get(c.data_type) == Type.TYPE_VECTOR:
+            data_entity_type = DataEntityType.VECTOR_STORE
+            break
```

Comment:
This code block and the block below are literally the same. The only difference is default value for `data_enitty_type` variable. I would suggest to move it into some different python module like `utils.py` inside `mappers` directory and put that code there.

Something like:
```
def has_vector_column(columns):
  return next((True for column in columns if TYPES_SQL_TO_ODD.get(column.data_type) == Type.TYPE_VECTOR), False)
```
  
# I'm just trying to avoid nested blocks but for/if construction is more readable I guess

Reply:
@AndreyNenashev done.